data_models/models.py

```diff
@@ -56,6 +61,22 @@ class Meta:
 
     categories = models.ManyToManyField(Category, blank=True, verbose_name='Kategorier')
 
+    digas_id = models.IntegerField(verbose_name='Tilhørende Digas-program', unique=True, null=True)
+
+    is_podcast = models.BooleanField(
+        verbose_name='Programmet er en podkast',
+        default=False,
+        help_text='Podkaster får lenke til podkast-feeden, og (etterhvert) mulighet til å legge '
+        'inn podkast-episoder.<br/>Flere program kan markeres som podkast samtidig i '
```

Comment:
I was not sure whether to point it out to the user or not. I ultimately decided to include it in order to avoid a situation where someone manually goes into every single program and updates the podcast flag that way. If the people who probably will end up doing that job are made aware of this possibility, I'm fine with removing it. That will probably work out better too.

I don't think there's any quick way of making the Django admin actions more discoverable by users. But I think it can help to create more useful actions where possible, so that users learn to consult the dropdown (since they would see it provides useful shortcuts) rather than ignore it (because it currently serves no purpose). It could then be a useful feature which talents learn from each other.